druid-shell/src/platform/mac/window.rs

```diff
@@ -133,14 +137,18 @@ enum IdleKind {
 }
 
 /// This is the state associated with our custom NSView.
-struct ViewState {
+pub(crate) struct ViewState {
     nsview: WeakPtr,
-    handler: Box<dyn WinHandler>,
+    pub(crate) handler: Box<dyn WinHandler>,
     idle_queue: Arc<Mutex<Vec<IdleKind>>>,
     /// Tracks window focusing left clicks
     focus_click: bool,
     // Tracks whether we have already received the mouseExited event
     mouse_left: bool,
+    /// Pending alert requests
+    alert_queue: RefCell<VecDeque<AlertRequest>>,
```

Comment:
Is it correct that this is a queue? I'd have expected a stack (i.e. last alert fired is the first one to be dismissed)

Reply:
Alert dismissing order can be arbitrary.